Jenkinsfile_nightly

```diff
@@ -23,7 +23,9 @@ withNightlyPipeline(type, product, component) {
 
   loadVaultSecrets([
     'reform-scan-aat': [
-      secret('test-signing-key-der-base64', 'SIGNING_KEY_DER_BASE64')
+      secret('test-signing-key-der-base64', 'SIGNING_KEY_DER_BASE64'),
+      secret('idam-users-bulkscan-username', 'IDAM_USERS_BULKSCAN_USERNAME'),
+      secret('idam-users-bulkscan-password', 'IDAM_USERS_BULKSCAN_PASSWORD')
```

Comment:
i doubt `reform-scan` vault is aware of those secrets. they are in `bulk-scan` domain - perhaps we need to retrieve from there instead

Reply:
Yes, the secrets are in `bulk-scan`. Should we just copy them from there?

Reply:
nah, jenkins has access to them so we can just glue together different vaults (maps) in here with a set of secrets we need from them :wink: